tests/JavascriptGeneratorBundle/Fixtures/raml/platform-version/api.raml

```diff
@@ -0,0 +1,4 @@
+#%RAML 1.0
+title: Platform version override
+version: "1.0"
```

Comment:
Are quotes necessary? Plain `1.0` looks simpler, also quotes are not used on other properties with string value, eg. title

Reply:
They are necessary, because otherwise yaml parser will assume that it's a number and will convert it to "1", without the minor version